### ISSUE TITLE
Can we default to allow it only on chat.openai.com and google.com domains?

### ISSUE THREAD

**Issue by gastonmorixe** [qunash/chatgpt-advanced#149]
It works fine for me and feels a little more secure. 

Thank you

**Comment by kickymcassington** [comment]
Does this mean it will no longer "read and change all data on all webpages" ? 
I'm sure you all have good intentions but that is keeping me from installing it.
Looks so useful and awesome but every-time I've talked myself into installing this i see that permission and i get 2nd thoughts.

Thanks!

**Comment by gastonmorixe** [comment]
@kickymcassington yes, you can download it and just set it your self but it should be the default.

**Comment by kickymcassington** [comment]
Pardon if I'm mis-understanding. Sounds like I could limit its access after install or through the manual install.

I was referring to the extension link install method, I've tried both edge and firefox and both still say the extension will have access to all pages and all content in the browser. That seems like a bad idea for someone like me without fully trusting all the authors and unfortunately I'm not technical enough to go through the code myself. 
I was hoping this change would make it over to those only asking for more limited permissions and would lead to a safer install for less technical people like myself.

**Comment by qunash** [comment]
> @kickymcassington yes, you can install it and just set it your-self in the extension's settings in 10 seconds but it should be the default. Litte weird it's not.

That's not possible because chromium extensions do not support dynamic permission requests. i.e. an extension cannot ask for additional permissions after it's installed.
This permission is required to access info from any website, which is the core functionality of the extension.

**Comment by gastonmorixe** [comment]
> > @kickymcassington yes, you can install it and just set it your-self in the extension's settings in 10 seconds but it should be the default. Litte weird it's not.
> 
> That's not possible because chromium extensions do not support dynamic permission requests. i.e. an extension cannot ask for additional permissions after it's installed. This permission is required to access info from any website, which is the core functionality of the extension.

It works fine for me when I only allow Google and chat.openai.com

**Comment by qunash** [comment]
You can fork the repo, restrict permissions and build it manually.

**Comment by gastonmorixe** [comment]
There's no need to fork, you can set google.com and chat.openai.com in settings
